git-annex-remote-googledrive

```diff
@@ -394,13 +395,15 @@ or re-run 'git-annex-remote-googledrive setup' followed by 'git annex enableremo
         else:
             raise RemoteError ("There are two or more files named {}".format(key))
     
-    def _getfolder(self, path, root=None, create=True):
+    def _getfolder(self, path, root=None, root_id=None, create=True):
```

Comment:
I would like to only use file objects to refer to files and folders whereever possible. The `root` parameter already does what `root_id` is supposed to do, only it takes a file object. I don't think we should have a second parameter for the same purpose. Also, it's only ever going to be used once in this context, so I see no problem in creating the file object before running `_getfolder`.